dnsapi/dns_one.sh

```diff
@@ -4,48 +4,68 @@
 # one.com ui wrapper for acme.sh
 # Author: github: @diseq
 # Created: 2019-02-17
+# Fixed by: @der-berni
+# Modified: 2019-05-20
 #
-#     export ONECOM_User="username"
-#     export ONECOM_Password="password"
+#     export ONECOM_USER="username"
```

Comment:
do not change the name from  `ONECOM_User`  to  `ONECOM_USER`.
The existing users will kill you.

Reply:
That was my fault. I misread.
https://github.com/Neilpang/acme.sh/pull/2098#discussion_r258946332

See my second commit.